app/controllers/v1/post911_gi_bill_statuses_controller.rb

```diff
@@ -19,6 +19,8 @@ def show
       response = service.get_gi_bill_status
       render json: response,
              serializer: Post911GIBillStatusSerializer
+    rescue Breakers::OutageException => e
+      raise e
```

Comment:
The `OutageException` will bubble up from the `service`, which we again raise so that `ApplicationController` can handle it (finally returning a `ServiceOutage`)